tests/test_datasets/test_internal_datasets.py

```diff
@@ -126,6 +126,34 @@ def test_not_present_part():
             pd.to_datetime("2022-01-01 00:00:00"),
             ("train", "test"),
         ),
+        (
+            "m3_monthly",
+            (144, 2856),
```

Comment:
Let's replace the number of rows by the sum of all parts, as in other datasets, e.g. for "m3_monthly" (126 + 18, 2856).